app/containers/SearchTrackContainer/TrackDetailsContainer/index.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ * TrackDetailsContainer Container
+ *
+ */
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { injectIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import { createStructuredSelector } from 'reselect';
+import { injectSaga } from 'redux-injectors';
+import styled from 'styled-components';
+import { selectTrackDetails, selectTrackDetailsError, selectSearchTrackContainer } from '../selectors';
+import searchTrackContainerSaga from '../saga';
+import { searchTrackContainerCreators } from '../reducer';
+import TracksCard from '@app/components/TracksCard/index';
+import { colors } from '@themes';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const CustomDetailsContainer = styled.div`
+  background-color: ${colors.secondary};
+  border-radius: 0.5rem;
+  width: 32rem;
+  display: flex;
+  gap: 2rem;
+`;
+
+const CustomExtraDetailsContainer = styled.div`
```

Comment:
CustomTrackDetailContainer will be better

Reply:
this container is for displaying the extra details than that of the card that's why I did that
 but if you think that is better naming convention than I will change